samples/subsys/ipc/ipc_service/icmsg/remote/src/main.c

```diff
@@ -53,19 +76,33 @@ static int send_for_time(struct ipc_ept *ep, const int64_t sending_time_ms)
 	size_t bytes_sent = 0;
 	int ret = 0;
 
+#if defined(CONFIG_MULTITHREADING)
 	LOG_INF("Perform sends for %lld [ms]", sending_time_ms);
+#else
+	printk("Perform sends for %lld [ms]\r\n", sending_time_ms);
```

Comment:
How about a macro that resolves into LOG or printk+\r\n depending on multithreading?